### PR TITLE
Add Clean Reinstall Instructions

### LINKED ISSUE #92: VS Code Nuclear Option
VS Code leaves files in many places: project, `~/.vscode`, system standard configuration location in home directory, et al.  I've run into some situations where the REAL way to restart is to delete all VS Code data.  We should include this in the Restart section.  Maybe call it "delete all configuration and extensions" or something.

Here's the Microsoft docs: https://code.visualstudio.com/docs/setup/uninstall#_clean-uninstall

I had to do this on multiple machines over the last week.

### PR DESCRIPTION
**Pull request by ohjuny**
Closes #92 
Added Reinstall instructions within Restart section.

Not 100% sure about the PowerShell commands, in particular whether the `%APPDATA%` and `%USERPROFILE%` aliases can be used literally. Do not have access to a Windows machine to test this.

### CODE REVIEW

**Review by awdeorio — APPROVED** [review]
LGTM!  I made some changes.  Before merging, I'd like to get somebody with a Windows machine to verify that I got the the paths right for WSL @melodell might be able to help?

File: docs/setup_vscode.md [hunk 1]
```diff
@@ -43,25 +43,19 @@ $ cp -a p1-stats p1-stats.bak  # Backup
...
-
-#### macOS
-To uninstall VSCode, open Terminal and run the following commands.
+Next, remove all VS Code configuration files and extensions (based on [Microsoft instructions](https://code.visualstudio.com/docs/setup/uninstall#_clean-uninstall)).
 
+**macOS:**
 ```console
-$ rm -rf ./Library/Application Support/Code
 $ rm -rf ~/.vscode
+$ rm -rf ~/Library/Application\ Support/Code
```
Comment (ohjuny) [inline docs/setup_vscode.md:52]: Good catch!

**Comment by melodell** [comment]
> I'd like to get somebody with a Windows machine to verify that I got the the paths right for WSL @melodell might be able to help?

Not a Windows person nor do I have access to one 😔

File: docs/setup_vscode.md [hunk 2]
```diff
@@ -43,6 +43,22 @@ $ cp -a p1-stats p1-stats.bak  # Backup
...
+```console
+$ rm -rf ~/.vscode
+$ rm -rf ~/Library/Application\ Support/Code
+```
+
+**Windows:**  Be sure to use your Windows username.
+```console
+$ rm -rf ~/.vscode
+$ rm -rf /mnt/c/<windows username>/.vscode
+$ rm -rf /mn/c/<windows username>/AppData/Code
```
Comment (melodell) [inline docs/setup_vscode.md:59]:
```suggestion
$ rm -rf /mnt/c/<windows username>/AppData/Code
```

File: docs/setup_vscode.md [hunk 3]
```diff
@@ -43,6 +43,22 @@ $ cp -a p1-stats p1-stats.bak  # Backup
...
+**macOS:**
+```console
+$ rm -rf ~/.vscode
+$ rm -rf ~/Library/Application\ Support/Code
+```
+
+**Windows:**  Be sure to use your Windows username.
+```console
+$ rm -rf ~/.vscode
+$ rm -rf /mnt/c/<windows username>/.vscode
```
Comment (seshrs) [inline docs/setup_vscode.md:58]: Hi! 👋

It’s actually:
```
/mnt/c/Users/<username>/.vscode
/mnt/c/Users/<username>/AppData/Roaming/Code
```

**Comment by seshrs** [comment]
> Not 100% sure about the PowerShell commands, in particular whether the %APPDATA% and %USERPROFILE% aliases can be used literally.

Will students be using Powershell? Then they can use:
```
PS > rm -rf "$env:APPDATA/Code"
```

If they'll be using WSL, then they can use:
```console
$ APPDATA=`cmd.exe /c echo %APPDATA% 2> /dev/null | tr -d '\r' | xargs -0 wslpath`
$ echo $APPDATA
/mnt/c/Users/sesh/AppData/Roaming
$ rm -rf $APPDATA/Code
```

**Comment by awdeorio** [comment]
Thanks for taking a look @seshrs !  A couple questions:

Is `wslpath` installed by default or would a student need to install a separate package?

Does `wslpath` have a simpler invocation, something like `wslpath env | grep APPDATA` or something?

**Comment by seshrs** [comment]
(Sorry I haven’t had a chance to reply yet, since I’m traveling for the next week. I think you’re right that this should be extra table without using `wslpath`. I’ll check this out again next week if you don’t get a chance to try it on a Windows laptop :) )

**Comment by awdeorio** [comment]
Checking in @seshrs .  Thanks for your comments!

**Comment by seshrs** [comment]
Sorry I didn’t return to this! Unfortunately I’m… uh… traveling again (😂) and won’t be near my Windows PC for another week lol

Looking at the command, I don’t think you really need `wslpath` in this invocation since it only appears to convert a Windows path to a WSL-compatible path. 

In other words, almost all Windows users have the app data directory at `/mnt/c/Users/<username>/AppData/Roaming`. So maybe most people won’t need the programmatic solution to get the directory path. 

---
If the question is simply about avoiding `xargs`, we could always direct students to copy the output from the CMD command, and paste it as an argument to `wslpath` :)

**Comment by awdeorio** [comment]
Thanks for the confirmation @seshrs !  I think this is good to go.

**Pull request merged**